webapp/src/components/InputField/index.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import {InputFieldType} from 'types';
+
+type Props = {
+    heading: {
+        id: string;
+        message: string;
+    };
```

Comment:
```suggestion
    heading: string | ReactNode;
```
Refer https://github.com/mattermost/mattermost-webapp/blob/7c09150efdca6e488efedadea793b2b681822a48/components/quick_input/quick_input.tsx#L43